multinet/user.py

```diff
@@ -1,163 +1,204 @@
 """User data and functions."""
+from __future__ import annotations  # noqa: T484
```

Comment:
What's this all about?

Reply:
If you have a class method that returns an instance of that class, and you want to type annotate it properly, this is required. In this case, I believe they're all static methods. Here's an example https://github.com/multinet-app/multinet-server/blob/550d608f6d0dc7452d35672b4edb3d94ea3969e0/multinet/user.py#L92-L100